### PR TITLE
Re cleanedup

### PR DESCRIPTION
**Pull request by medariox**
**_ISSUES_**
- ~~Groups are all lowercase~~, missing groups are empty instead of "None".

- ~~Refresh during manual search doesn't work.~~

- ~~"The" is not ignored during name sorting.*~~
image

- ~~Can't change "sort by" order.*~~

- ~~"Sort by" setting is not saved.~~

~~* Only on "initial state" - changing "direction" fixes this until page refresh. Following JS errors:~~
image

### CODE REVIEW

**Comment by fernandog** [comment]
@medariox @labrys 

1) Options are under tab name - It should be in the right "div"
/config/notifications/
/config/general/
/config/backuprestore/
/config/anime/

2) size in failed downloads are not pretty: 3706011067  (maybe this also exists in devleop)

3) Manual search results page it not being refreshed with results

File: gui/slick/views/addShows_popularShows.mako [hunk 1]
```diff
@@ -35,25 +35,19 @@
 <br>
 <div id="popularShows">
     <div id="container">
-    % if not popular_shows:
+% if not popular_shows:
```
Comment (fernandog) [inline gui/slick/views/addShows_popularShows.mako]: @medariox i would keep the space here as this code belongs to the `<div id="container">`

File: gui/slick/views/home.mako [hunk 2]
```diff
@@ -44,27 +44,27 @@
...
-                                <option value="network" data-sort="${srRoot}/setPosterSortBy/?sort=network" ${('', 'selected="selected"')[sickbeard.POSTER_SORTBY == 'network']}>Network</option>
-                                <option value="progress" data-sort="${srRoot}/setPosterSortBy/?sort=progress" ${('', 'selected="selected"')[sickbeard.POSTER_SORTBY == 'progress']}>Progress</option>
+                                <option value="name" data-sort="/setPosterSortBy/?sort=name" ${('', 'selected="selected"')[sickbeard.POSTER_SORTBY == 'name']}>Name</option>
+                                <option value="date" data-sort="/setPosterSortBy/?sort=date" ${('', 'selected="selected"')[sickbeard.POSTER_SORTBY == 'date']}>Next Episode</option>
+                                <option value="network" data-sort="/setPosterSortBy/?sort=network" ${('', 'selected="selected"')[sickbeard.POSTER_SORTBY == 'network']}>Network</option>
+                                <option value="progress" data-sort="/setPosterSortBy/?sort=progress" ${('', 'selected="selected"')[sickbeard.POSTER_SORTBY == 'progress']}>Progress</option>
```
Comment (fernandog) [inline gui/slick/views/home.mako]: @medariox @labrys href doesnt work here, right?

File: gui/slick/views/history.mako [hunk 3]
```diff
@@ -38,8 +38,8 @@
 
     <span> Layout:
         <select name="HistoryLayout" class="form-control form-control-inline input-sm" onchange="location = this.options[this.selectedIndex].value;">
-            <option value="${srRoot}/setHistoryLayout/?layout=compact"  ${('', 'selected="selected"')[sickbeard.HISTORY_LAYOUT == 'compact']}>Compact</option>
-            <option value="${srRoot}/setHistoryLayout/?layout=detailed" ${('', 'selected="selected"')[sickbeard.HISTORY_LAYOUT == 'detailed']}>Detailed</option>
+            <option value="/setHistoryLayout/?layout=compact"  ${('', 'selected="selected"')[sickbeard.HISTORY_LAYOUT == 'compact']}>Compact</option>
+            <option value="/setHistoryLayout/?layout=detailed" ${('', 'selected="selected"')[sickbeard.HISTORY_LAYOUT == 'detailed']}>Detailed</option>
```
Comment (fernandog) [inline gui/slick/views/history.mako]: @medariox @labrys also here (href)

File: gui/slick/views/schedule.mako [hunk 4]
```diff
@@ -26,28 +26,28 @@
...
-            <option value="${srRoot}/setScheduleLayout/?layout=calendar" ${('', 'selected="selected"')[sickbeard.COMING_EPS_LAYOUT == 'calendar']} >Calendar</option>
-            <option value="${srRoot}/setScheduleLayout/?layout=banner" ${('', 'selected="selected"')[sickbeard.COMING_EPS_LAYOUT == 'banner']} >Banner</option>
-            <option value="${srRoot}/setScheduleLayout/?layout=list" ${('', 'selected="selected"')[sickbeard.COMING_EPS_LAYOUT == 'list']} >List</option>
+            <option value="/setScheduleLayout/?layout=poster" ${('', 'selected="selected"')[sickbeard.COMING_EPS_LAYOUT == 'poster']} >Poster</option>
+            <option value="/setScheduleLayout/?layout=calendar" ${('', 'selected="selected"')[sickbeard.COMING_EPS_LAYOUT == 'calendar']} >Calendar</option>
+            <option value="/setScheduleLayout/?layout=banner" ${('', 'selected="selected"')[sickbeard.COMING_EPS_LAYOUT == 'banner']} >Banner</option>
+            <option value="/setScheduleLayout/?layout=list" ${('', 'selected="selected"')[sickbeard.COMING_EPS_LAYOUT == 'list']} >List</option>
```
Comment (fernandog) [inline gui/slick/views/schedule.mako]: @medariox @labrys also here (href)

File: gui/slick/views/home.mako [hunk 5]
```diff
@@ -44,27 +44,27 @@
...
                             </select>
                         </span>
 
                         <span class="show-option"> Direction:
                             <select id="postersortdirection" class="form-control form-control-inline input-sm">
-                                <option value="true" data-sort="${srRoot}/setPosterSortDir/?direction=1" ${('', 'selected="selected"')[sickbeard.POSTER_SORTDIR == 1]}>Ascending </option>
-                                <option value="false" data-sort="${srRoot}/setPosterSortDir/?direction=0" ${('', 'selected="selected"')[sickbeard.POSTER_SORTDIR == 0]}>Descending</option>
+                                <option value="true" data-sort="/setPosterSortDir/?direction=1" ${('', 'selected="selected"')[sickbeard.POSTER_SORTDIR == 1]}>Ascending </option>
+                                <option value="false" data-sort="/setPosterSortDir/?direction=0" ${('', 'selected="selected"')[sickbeard.POSTER_SORTDIR == 0]}>Descending</option>
```
Comment (fernandog) [inline gui/slick/views/home.mako]: @medariox @labrys also here (href)

File: gui/slick/views/home.mako [hunk 6]
```diff
@@ -44,27 +44,27 @@
...
-                            <option value="${srRoot}/setHomeLayout/?layout=small" ${('', 'selected="selected"')[sickbeard.HOME_LAYOUT == 'small']}>Small Poster</option>
-                            <option value="${srRoot}/setHomeLayout/?layout=banner" ${('', 'selected="selected"')[sickbeard.HOME_LAYOUT == 'banner']}>Banner</option>
-                            <option value="${srRoot}/setHomeLayout/?layout=simple" ${('', 'selected="selected"')[sickbeard.HOME_LAYOUT == 'simple']}>Simple</option>
+                            <option value="/setHomeLayout/?layout=poster" ${('', 'selected="selected"')[sickbeard.HOME_LAYOUT == 'poster']}>Poster</option>
+                            <option value="/setHomeLayout/?layout=small" ${('', 'selected="selected"')[sickbeard.HOME_LAYOUT == 'small']}>Small Poster</option>
+                            <option value="/setHomeLayout/?layout=banner" ${('', 'selected="selected"')[sickbeard.HOME_LAYOUT == 'banner']}>Banner</option>
+                            <option value="/setHomeLayout/?layout=simple" ${('', 'selected="selected"')[sickbeard.HOME_LAYOUT == 'simple']}>Simple</option>
```
Comment (fernandog) [inline gui/slick/views/home.mako]: @medariox @labrys also here (href)

**Comment by fernandog** [comment]
Approved

**Comment by fernandog** [comment]
@medariox @Thraxis 

1) Size of second "select" is wrong
2) Font of the "selects" its not the same as the UI

image

**Comment by Thraxis** [comment]
This is what happens when you remove the srRoot from all the CSS, JS and Mako files and pymedusa is configured to run with a webroot:

image

**Comment by Thraxis** [comment]
To clarify my last coment.

base href works with "PATH" relative URL's

URL's that start with **/** are "DOCUMENT ROOT" relative

for the base href to work correctly all URL's that had a srRoot need to now have their leading / removed as well.

**Comment by medariox** [comment]
Yes, I'm currently changing that. Thanks for testing @Thraxis

**Comment by fernandog** [comment]
What about viewlogs? I have no clue how to fix it

**Comment by fernandog** [comment]
Approved

**Comment by Thraxis** [comment]
running with a webroot looks good

**Pull request merged**